test/suites/zombie_tanssi_relay_eth_bridge/test_zombie_tanssi_relay_eth_bridge.ts

```diff
@@ -423,6 +453,85 @@ describeSuite({
 
         it({
             id: "T06",
+            title: "Rewards are claimable",
+            test: async () => {
+                // Find the first era index claimable
+                const currentEra = (await relayApi.query.externalValidators.activeEra()).unwrap().index;
+
+                let eraToAnalyze = currentEra.toNumber();
+                console.log(await operatorRewardContract.eraRoot(eraToAnalyze));
+
+                // Try to find latest reported era
+                // eraRoot returns a struct of 4 items, the 3rd of which is the era root
+                // we try to retrieve the first non-default one
+                while (
+                    (await operatorRewardContract.eraRoot(eraToAnalyze))[3] ===
+                        "0x0000000000000000000000000000000000000000000000000000000000000000" &&
+                    eraToAnalyze >= 0
```

Comment:
It would be great if we keep named constants instead of values like this. So it will be clear what it is about. 
So instead of 
```
(await operatorRewardContract.eraRoot(eraToAnalyze))[3] === "0x0000000000000000000000000000000000000000000000000000000000000000"
```
we would have something like:
```
const ERA_ROOT_ADDRESS = "0x0000000000000000000000000000000000000000000000000000000000000000"; // or whatever
(await operatorRewardContract.eraRoot(eraToAnalyze))[3] === ERA_ROOT_ADDRESS

```

Reply:
DEFAULT_ERA_ROOT_ADDRESS good?